share/jupyter/nbconvert/templates/reveal/index.html

```diff
@@ -1,5 +1,5 @@
-{%- extends 'basic.tpl' -%}
```

Comment:
does `default` stand for the default html template?

Reply:
yes, I think we can make reveal actually a normal template, just like we want with voila, with just some extra resources. So you would do `--template reveal`.